codec/codec_test.go

```diff
@@ -356,7 +355,9 @@ func setUpWithServer(t *testing.T, version string, server *httptest.Server, with
 }
 
 func fromFile(t *testing.T) []byte {
-	path := filepath.Join("testdata", t.Name())
+	currentPath, err := os.Getwd()
```

Comment:
`path := filepath.Join("./testdata", t.Name())` should do.

I would recommend against using `os.Getwd()`. Go will run the tests in a way that `testdata` will be relative to the actual test.